srcpkgs/intel-parallel-studio-xe/files/EULA.txt

```diff
@@ -0,0 +1,274 @@
+IMPORTANT - READ BEFORE COPYING, INSTALLING OR USING.
```

Comment:
License should be downloaded with distfiles to not become stale.

Reply:
This shouldn't have been there. Thanks!